instrumentation/opentelemetry-api-beta/src/main/java/io/opentelemetry/auto/instrumentation/opentelemetryapi/trace/Bridging.java

```diff
@@ -100,17 +101,20 @@ public static SpanContext toUnshaded(final io.opentelemetry.trace.SpanContext sh
     }
   }
 
-  public static Map<String, io.opentelemetry.common.AttributeValue> toShaded(
-      final Map<String, AttributeValue> unshadedAttributes) {
-    final Map<String, io.opentelemetry.common.AttributeValue> shadedAttributes = new HashMap<>();
-    for (final Map.Entry<String, AttributeValue> entry : unshadedAttributes.entrySet()) {
-      final AttributeValue value = entry.getValue();
-      final io.opentelemetry.common.AttributeValue shadedValue = toShadedOrNull(value);
-      if (shadedValue != null) {
-        shadedAttributes.put(entry.getKey(), shadedValue);
-      }
-    }
-    return shadedAttributes;
+  public static io.opentelemetry.common.Attributes toShaded(final Attributes unshadedAttributes) {
```

Comment:
(in the future) this probably is better in common package, like you did for LabelsShader